frontend/scripts/react-components/home/home.component.jsx

```diff
@@ -1,10 +1,8 @@
-import React, { useEffect } from 'react';
+import React, { Suspense, useEffect } from 'react';
```

Comment:
About Suspense, it's still experimental as a data fetching and suspending UI. However, for code-splitting using lazy it's already stable (if they change it, it will have to be backwards compatible).